README.md

```diff
@@ -1,2 +1,84 @@
-# charts
-Contane Helm Charts
+# Contane Helm Charts
+Welcome to the Contane Helm Charts repository! Here, you'll find a collection of Helm Charts for all the open-source projects developed by Contane. These charts are designed to simplify the deployment and management of applications on Kubernetes clusters.
```

Comment:
This is slightly inaccurate - we cannot provide a Helm chart for *all* open-source projects, given that not all projects are applications (e.g., docker-yamllint). I also think we can make it more concise.

```suggestion
A collection of Helm charts for open-source applications developed by Contane, designed to simplify the deployment on Kubernetes.
```